magento/inventory
# MSI-908: Separate UI and Business logic for InventoryCatalog Module.

### Description
Separate UI and Business logic for InventoryCatalog Module
Contains changes from magento-engcom/msi#907

### Fixed Issues (if relevant)
1. magento-engcom/msi#908: Separate UI and Business logic for InventoryCatalog Module.

### Manual testing scenarios
1. Make sure you have no additional stock and sources.
2. Login to admin area.
3. Start creating simple product.
4. Check there are no additional UI elements related to MSI on product edit page.
5. Create additional source and stock.
6. Start creating simple product.
7. Check addition UI element like: "Sources" tab appear with buttons "Assign Sources" and "Advanced Inventory".
8. Try to add, remove and edit sources. Check there are no errors.
9. Check "Advanced Inventory" modal window appear after "Advanced Inventory" button been pressed.
10, Check it contains all necessary information.


### Contribution checklist
